quickstarts/deploy.markdown

```diff
@@ -0,0 +1,59 @@
+Kroxylicious is a Java application based on [Netty](https://netty.io/), which means it will run anywhere you can run a JVM. (That's a lot of places!)
```

Comment:
rather than deploy.markdown, can we use `.md` to match the convention we have elsewhere.

Reply:
I've been following the convention used by Jekyll, which is to use `.markdown` for pages and site content, and `.md` for things like the README that should not be rendered with the rest of the website. There is functionally no difference between using one or the other and I'm happy to change it if that's what we would prefer, but I personally like the distinction it draws between utility files and actual content.